pkg/net/libp2p/authenticated_connection.go

```diff
@@ -146,12 +146,12 @@ func (ac *authenticatedConnection) runHandshakeAsInitiator() error {
 	// Act 1
 	//
 
-	initiatorAct1, err := handshake.InitiateHandshake(ac.protocol)
+	InitiatorAct1, err := handshake.InitiateHandshake(ac.protocol)
```

Comment:
Why do we capitalize the variable name as well (here and all changes below)? Even if we need to export `InitiatorAct*` etc., we should leave all variables names uncapitalized. This is the convention in Go.

Reply:
Sorry, my bad, fixed in 553773d9cb7555c8ef5ed6fde47829718774f59d.